src/v/utils/tests/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ rp_test(
     object_pool_test.cc
     delta_for_test.cc
     token_bucket_test.cc
+    utf8_control_chars.cc
```

Comment:
I think this should go in `utils_unit` test, or it is ignored (either that or it disables all the seastar ones, can't remember which)

Reply:
Not resolved?

Reply:
Will be!  Just need to push it (sorry I mark things resolved as I solve them before I push them).